common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -191,20 +203,23 @@ object LogKeys {
   case object HIVE_OPERATION_TYPE extends LogKey
   case object HOST extends LogKey
   case object HOST_PORT extends LogKey
-  case object IDENTIFIER extends LogKey
```

Comment:
Unified as` LogKeys.TABLE_NAME`, remove `LogKeys.IDENTIFIER`

Reply:
Currently, only `two places` use this `key`, and they all mean `the name of the table`.

Reply:
Thanks for unifying them!